### ISSUE TITLE
How to train on R4R

### ISSUE THREAD

**Issue by DongMasarki** [YicongHong/Entity-Graph-VLN#2]
Hi, I'm very interesting in your work. I want to know how you trained your model on R4R dataset? Did you use the "goal reward" or "fidelity reward", and could your please share your code for training on R4R?

**Comment by YicongHong** [comment]
Hi, training on R4R is the same as the training on R2R, except setting the maximum length of input instructions to 120. We only use the distance reward as in [EnvDrop](https://arxiv.org/abs/1904.04195).

Cheers.

**Comment by DongMasarki** [comment]
Thanks for your reply, I found traning on R4R is very time-consuming, do u have any idea?

**Comment by YicongHong** [comment]
Yeah the training should takes longer due to the much greater number of training samples, and each sample has a longer path. But in our work, we only performed stage 1 training for the R4R (w/o using a Speaker), so the overall time should be less than for R2R.

Cheers.

**Comment by DongMasarki** [comment]
Thanks for your reply, may I ask the last question? When training on R4R, did you set the batchsize=64 and train it for 80000 iterations?

Cheers,

**Comment by YicongHong** [comment]
No worries. Thanks for reminding me these hyperparameters, we set BatchSize=32 and train the network for 100,000 iterations.

**Comment by DongMasarki** [comment]
Thanks for your generous sharing!

Best regards,
